extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1174,11 +1178,27 @@ export class AmpA4A extends AMP.BaseElement {
             'src': srcPath,
             'name': nameData,
           }, SHARED_IFRAME_PROPERTIES));
+      if (method == XORIGIN_MODE.NAMEFRAME) {
+        const attributes = this.generateSentinelAndContext(iframe);
```

Comment:
I've been using it somewhat interchangeably, but sentinel and context together are the attributes assigned on the frame. 'context' is referring to the old context that did not include sentinel